### ISSUE TITLE
Import-BcContainerLicense fails when using a url.

### ISSUE THREAD

**Issue by mjmatthiesen** [microsoft/navcontainerhelper#2533]
**Describe the issue**
BC licenses have silly short expiry time frames, so I need to up date the license. I destroy containers for new version, but depending on expiry dates, I need to occasionally update the license file. I know this used to work as I have used this command before, but at some point it broke.
Only thing I can note is that the repeated url on the download call does not include the query part of the url, i.e. ?=StuffAzureBlobsCreate

I should also note that I am able to access said url through any browser on any device and download the file no problem. If I use
Invoke-WebRequest -Uri $license -OutFile "C:\license.flf"
I can download the file no problem as well.

**Scripts used to create container and cause the issue**
```
$additionalParameters = @("--publish 8080:8080",
                          "--publish 443:443",
                          "--publish 7046-7049:7046-7049")
New-BCContainer `
-accept_eula `
-containerName bcserver `
-auth NavUserPassword `
-updateHosts `
-usebestcontaineros `
-assignPremiumPlan `
-licenseFile $license `
-artifactUrl $artifactUrl `
-Credential $credential `
-PublicDnsName $dnsname `
-installCertificateOnHost `
-useSSL `
-includeAl `
-additionalParameters $additionalParameters
```

=> Zero issues.

(month or two later)
```
$license = "https://wow.blob.core.windows.net/url/file?dataStuff"
Import-BcContainerLicense -containerName bcserver -licenseFile $license
```

For the output, I obviously no longer have the output of the create call, but this is on the later executed import call:
```
Downloading license file 'https://wow.blob.core.windows.net/url/file' to container
Import-BcContainerLicense Telemetry Correlation Id: a5ef7008-124f-4630-bc11-7c20f01e31c3
Exception calling "DownloadFile" with "2" argument(s): "An exception occurred during a WebClient request."
At C:\Program
Files\WindowsPowerShell\Modules\bccontainerhelper\3.0.10\ContainerHandling\Import-NavContainerLicense.ps1:31 char:9
+         (New-Object System.Net.WebClient).DownloadFile($licensefile,  ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : WebException
```

```
BcContainerHelper is version 3.0.10
BcContainerHelper is running as administrator
Host is Microsoft Windows 10.0.17763 Build 17763
Docker Client Version is 19.03.13
Docker Server Version is 19.03.13
```

**Additional context**
- does it happen all the time? Yes
- did it use to work? Yes

**Comment by freddydk** [comment]
bccontainerhelper deliberately removes the query part of the url when printing out to avoid compromising secrets.
You can use invoke-webrequest - can you also use the Download-File with the URL?
ex.
download-file -sourceurl $url -destinationfile "c:\temp\file.flf"

**Comment by mjmatthiesen** [comment]
Yes, that also works.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 3.0.11
